src/main/webapp/assets/css/indexStyle.css

```diff
@@ -62,20 +57,17 @@ select::-ms-expand {
 
 .selectdiv::after {
   color: #565656;
-  content: '🛆';
+  content: 'v';
   font: 20px "Consolas", monospace;
```

Comment:
How come the font type would be different from the main font? The size I totally understand for the arrow to look the same as the size of the picker of colleges!

Reply:
Good catch, I wasn't thinking :)

Reply:
Actually, upon further testing I'm going to leave the `v` monospace because it makes it more short and stout, which looks nicer as a dropdown arrow